profile/profile.yaml

```diff
@@ -0,0 +1,179 @@
+# Preferred name (Characters only)
+name: "Priya Nagarkar"
+# Your GitHub repo root URL
+github_repo_root_url: https://github.com/jarviscanada/jarvis_data_eng_Priya
+
+# Introduce yourself, your academic and professional background.
+summary: "I am a graduate from George Brown College. I have four years of technical working experience.
+I worked as an Junior Engineer, major job responsibilites include PLC-SCADA programming.
+I am working as an Data Engineer at Jarvis. I have familiarized myself with the core concepts of the subject and have gained valuable practice in using software that is prevalent in the industry.
+Furthermore, soft skills like interpersonal communication and diverse collaboration have been proven through various professional development experiences.
+I am seeking to utilize the practical and analytical knowledge developed in personal projects and academia to leverage a challenging career path in the field of data engineering I am a team-oriented person, and problem solver. I enjoy coding challenges and learning algorithms and data structures.
+ Currently, I am looking for a great team to provide my skills and learn new tech skills."
```

Comment:
you need to improve your writing. 

sample profiles

https://github.com/jarviscanada/jarvis_data_eng_mathew
https://github.com/jarviscanada/jarvis_data_eng_TuanMai
https://github.com/jarviscanada/jarvis_data_eng_DaydeReid